hubspot/management/commands/sync_hubspot.py

```diff
@@ -75,6 +77,24 @@ def sync_products(self):
         )
         print("  Finished")
 
+    def sync_b2b_deals(self):
```

Comment:
Will now need another method `sync_b2b_contacts` as well

Reply:
@mbertrand again, I think we are not syncing over the User model in B2B case.

Reply:
If we are syncing previous b2b deals, it seems we should sync the contacts (ie emails) they are associated with as well?  @pdpinch is that right?

Reply:
@mbertrand I did not get to this point over syncing b2b contacts while we may do not have user instance in our model. can you take a look to my last commit. is it make sense or anything else ?